integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Domain.java

```diff
@@ -986,37 +987,61 @@ private static void scaleViaScript(String opNamespace, String domainNamespace,
+        .append("https://$KUBERNETES_SERVICE_HOST:$KUBERNETES_SERVICE_PORT")
+        .append(" 2>> /u01/scalingAction.out ");
 
 
     String commandToExecuteInsidePod = scalingCommand.toString();
+    ExecResult result = null;
+    try {
+      result = assertDoesNotThrow(() -> Kubernetes.exec(adminPod, null, true,
+              "/bin/sh", "-c", commandToExecuteInsidePod),
+              String.format("Could not execute the command %s in pod %s, namespace %s",
+                      commandToExecuteInsidePod, adminPod.getMetadata().getName(), domainNamespace));
+      logger.info("Command {0} returned with exit value {1}, stderr {2}, stdout {3}",
+              commandToExecuteInsidePod, result.exitValue(), result.stderr(), result.stdout());
+    } catch (Error err) {
+      logger.info("Command {0} returned with exit value {1}, stderr {2}, stdout {3}",
+              commandToExecuteInsidePod, result.exitValue(), result.stderr(), result.stdout());
+      // copy scalingAction.log to local
+      testUntil(
+              () -> copyFileFromPod(domainNamespace, adminPod.getMetadata().getName(), null,
+                      "/u01/scalingAction.log",
+                      Paths.get(RESULTS_ROOT + "/" + domainUid + "-scalingAction.log")),
+              logger,
+              "Copying scalingAction.log from admin server pod");
+      // copy scalingAction.out to local
+      testUntil(
+              () -> copyFileFromPod(domainNamespace, adminPod.getMetadata().getName(), null,
+                      "/u01/scalingAction.out",
+                      Paths.get(RESULTS_ROOT + "/" + domainUid + "-scalingAction.out")),
+              logger,
+              "Copying scalingAction.out from admin server pod");
+      throw err;
 
-    ExecResult result = assertDoesNotThrow(() -> Kubernetes.exec(adminPod, null, true,
-        "/bin/sh", "-c", commandToExecuteInsidePod),
-        String.format("Could not execute the command %s in pod %s, namespace %s",
-            commandToExecuteInsidePod, adminPod.getMetadata().getName(), domainNamespace));
-    logger.info("Command {0} returned with exit value {1}, stderr {2}, stdout {3}",
-        commandToExecuteInsidePod, result.exitValue(), result.stderr(), result.stdout());
-
+    }
     // copy scalingAction.log to local
-    withStandardRetryPolicy
-        .conditionEvaluationListener(
-            condition -> logger.info("Copying scalingAction.log from admin server pod, waiting for success "
-                    + "(elapsed time {0}ms, remaining time {1}ms)",
-                condition.getElapsedTimeInMS(),
-                condition.getRemainingTimeInMS()))
-        .until(() -> {
-          return copyFileFromPod(domainNamespace, adminPod.getMetadata().getName(), null,
-              domainHomeLocation + "/bin/scripts/scalingAction.log",
-              Paths.get(RESULTS_ROOT + "/" + domainUid + "-scalingAction.log"));
-        });
+    testUntil(
```

Comment:
the following section for copying scalingAction.log and scalingAction.out seems the same as above.

Reply:
replaced with private method